### ISSUE TITLE
Handle extra directory level created by bundled submission downloads

### ISSUE THREAD

**Issue by joshuathayer** [freedomofpress/securedrop-workstation#85]
See the discussion in #8 - 

Another oddity is that when downloading multiple files from securedrop, it puts each file in its own directory in the generated tarball. Our processing does not remove that directory level, so if (over time) a journalist downloads both single files and a bundle which contains those same files, they'll end up with a tree that might look like-

    Sources/monistic_convention/1-monistic_convention-doc
    Sources/monistic_convention/2-monistic_convention-msg
    Sources/monistic_convention/1_2017-09-22/1-monistic_convention-doc
    Sources/monistic_convention/2_2017-09-22/2-monistic_convention-msg

i.e., the same files are in two different places. I am not sure why SD adds that extra directory layer, but we should probably just remove that level in our processing.

@redshiftzero , I think I remember you ages ago saying there might be a change to how SD creates those submission zip files which would change the structure- does that ring a bell?

**Comment by redshiftzero** [comment]
Ah so - there was an issue on the SecureDrop server side where there was an unnecessary folder, but it was an additional source codename folder, i.e. in your example it would produce:

```
Sources/monistic_convention/monistic_convention/2_2017-09-22/2-monistic_convention-msg
```

This is issue https://github.com/freedomofpress/securedrop/issues/2262, and a fix for it was pushed out in SecureDrop 0.5 on December 5th, 2017. So the workstation-side fix in #6 (which resolved that issue) should no longer be necessary. But! you are totally correct that single file downloads are not wrapped in the folder that contains the date of the submission (as they are for the zip files - btw this was done upon request for journalists bulk downloading tons of documents at once). Workstation side, I think your suggestion of removing that (date folder) level in processing makes sense. We won't make any further changes server-side to the folder structure 😇

**Comment by redshiftzero** [comment]
Download logic now handled using `securedrop-client`
